<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>559</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="verticalTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="verticalTabWidgetPage1">
     <attribute name="title">
      <string>Pilotage manuel</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="moinschaudButton">
          <property name="text">
           <string>MOINS chaud</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="tempbare">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pluschaudButton">
          <property name="text">
           <string>PLUS chaud</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="bgButton">
          <property name="text">
           <string>GAUCHE/BAS</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="hButton">
          <property name="text">
           <string>HAUT</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="bButton">
          <property name="text">
           <string>BAS</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="hgButton">
          <property name="text">
           <string>GAUCHE/HAUT</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="hdButton">
          <property name="text">
           <string>DROITE/HAUT</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="bdButton">
          <property name="text">
           <string>DROITE/BAS</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="gButton">
          <property name="text">
           <string>GAUCHE</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="dButton">
          <property name="text">
           <string>DROITE</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="runstopbutton">
            <property name="text">
             <string>Run/Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="movewaylabel">
            <property name="text">
             <string>ordre</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuM">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuStatut">
    <property name="title">
     <string>Machine</string>
    </property>
    <addaction name="actionV_rifier_mahine"/>
    <addaction name="actionNouvelle"/>
    <addaction name="actionConnecter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionManuel"/>
    <addaction name="actionDebug"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuScreen_size">
     <property name="title">
      <string>Screen size</string>
     </property>
     <addaction name="actionLittle_height_1024px"/>
     <addaction name="actionNormal_15_fullscreen_test"/>
     <addaction name="actionAutodetect"/>
    </widget>
    <addaction name="actionLibrary_path"/>
    <addaction name="menuScreen_size"/>
    <addaction name="actionToggle_Library"/>
   </widget>
   <addaction name="menuM"/>
   <addaction name="menuStatut"/>
   <addaction name="menuAide"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionV_rifier_mahine">
   <property name="text">
    <string>VÃ©rifier machine</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionManuel">
   <property name="text">
    <string>Manuel</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionNouvelle">
   <property name="text">
    <string>Nouvelle...</string>
   </property>
  </action>
  <action name="actionConnecter">
   <property name="text">
    <string>Connecter...</string>
   </property>
  </action>
  <action name="actionLibrary_path">
   <property name="text">
    <string>Library path...</string>
   </property>
  </action>
  <action name="actionLittle_height_1024px">
   <property name="text">
    <string>Little(height&lt;960px)</string>
   </property>
  </action>
  <action name="actionNormal_15_fullscreen_test">
   <property name="text">
    <string>normal(15&quot; fullscreen test)</string>
   </property>
  </action>
  <action name="actionAutodetect">
   <property name="text">
    <string>Autodetect</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionToggle_Library">
   <property name="text">
    <string>Toggle Library</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
